lib/Steps/Integrations/Cordova.ts

```diff
@@ -179,6 +188,7 @@ export class Cordova extends BaseIntegration {
       [],
       'PBXShellScriptBuildPhase',
       'Upload Debug Symbols to Sentry',
+      // @ts-expect-error - xcode project type expects a string for the target param but we pass null (not sure why but this wasn't changed)
```

Comment:
@philprime I think you added types for `xcode` recently right? Any idea if this is an actual bug/misuse of the library on our end or a typing issue?

Reply:
Yes I did add the types but they were hand-made so there is a high probability that you should change the type to be nullable instead of the @ts-expect-error

Reply:
Thanks! I adjusted the `Project` type declaration ([c938286](https://github.com/getsentry/sentry-wizard/pull/881/commits/c93828679623946f210447e4d6521494c502d3da)). Turns out, the `@ts-expect-error` also muted another type error underneath which also required me to make a couple of the `optionsOrFolderType` props optional.